pom.xml

```diff
@@ -1816,7 +1874,26 @@
             </configuration>
          </plugin>
 
-
+         <plugin>
+             <groupId>org.wildfly.extras.batavia</groupId>
+             <artifactId>transformer-tools-mvn</artifactId>
+             <executions>
+                    <execution>
+                        <id>transform</id>
+                        <phase>process-classes</phase>
+                        <goals>
+                            <goal>transform-classes</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.wildfly.extras.batavia</groupId>
+                        <artifactId>transformer-impl-eclipse</artifactId>
+                        <version>${version.batavia}</version>
+                    </dependency>
+                </dependencies>
+         </plugin>
```

Comment:
Defining the execution here will mean the plugin runs (though doesn't necessarily do anything) for every module in the build. Is that needed, seems a bit ugly?

I dont recall it being that way last week when skimming the changes. If this came about to try and have the transformer-impl-eclipse dep version be consistent, just using the property in the previous places the version was specified would seem preferable if there isnt a specific reason this approach was needed.

Reply:
That way I don't have to redefine the dependency in each sub module

Reply:
Moved back to the initial build configuration